dogs.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/stylesheet.css" />
+    <title>Adopt-A-Dog</title>
+  </head>
+  <body>
+    <header>
+      <img class="logoimg" src="images/adoptadog-logo.svg" alt="adopt a dog logo"/>
+      <h1 class="headertext">Seattle's premiere dog adoption agency</h1>
+      <img class="cartimg" src="images/cart-icon.svg" alt="shopping cart icon" />
+      <form action="checkout.html">
+    <input type="submit" value="Checkout" />
+</form>
+    </header>
+
+    <div class="nav">
+      <ul>
+        <li><a href="index.html">Home /</a></li>
+        <li><a href="dogs.html">Dogs /</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </div>
+
+<div class="here">
+  <h2>Here are all of our dogs!</h2>
+</div>
+
+<article class="dogprofiles">
```

Comment:
I'd argue that the wrapping container is a section, and the dog tiles are more articles than sections (or a div)